### PR TITLE
Week3-Farzanna

### PR DESCRIPTION
**Pull request by Farzanna**

### CODE REVIEW

**Comment by Farzanna** [comment]
> Hi Farzanna, part 1 was not too difficult, was it? Your code looks fine and I approve this PR.

Hi Jim, thanks for review, yes part 2 was much more difficult than part 1, hope that part also be fine

File: homework-classes/App.js [hunk 1]
```diff
@@ -43,19 +50,24 @@ class App {
...
-      const contributors = await repo.fetchContributors();
+      const contributors = await repository.fetchContributors();
 
-      const repoContainer = Util.createAndAppend('div', this.mainContainer);
-      const contributorContainer = Util.createAndAppend('div', this.mainContainer);
+      const repoContainer = Util.createAndAppend('div', this.mainContainer, {
+        id: 'table-container',
+      });
+      const contributorContainer = Util.createAndAppend('div', this.mainContainer, {
+        class: 'Contributors-list',
```
Comment (remarcmij) [inline homework-classes/App.js]: Use **kebab-case** for CSS names (lowercase letters and hyphens): `contributors-list`.

File: homework-classes/App.js [hunk 2]
```diff
@@ -43,19 +50,24 @@ class App {
...
+      });
 
       const contributorList = Util.createAndAppend('ul', contributorContainer);
+      Util.createAndAppend('li', contributorList, { id: 'Contributors-h2', text: 'contributions' });
```
Comment (remarcmij) [inline homework-classes/App.js]: kebab-case

File: homework-classes/App.js [hunk 3]
```diff
@@ -70,7 +82,12 @@ class App {
    * @param {Error} error An Error object describing the error.
    */
   renderError(error) {
-    console.error(error); // TODO: replace with your own code
+    // console.error(error); // TODO: replace with your own code
```
Comment (remarcmij) [inline homework-classes/App.js]: Remove dead code (commented-out code).

**Comment by Farzanna** [comment]
> Hi @Farzanna, excellent work! But keep an eye on work-life balance. This is not important enough to lose sleep over it.
> 
> Good luck with the test and the rest of the curriculum.

Hi Jim, thanks for your feedback and your kind advice about making balance on life and work, it's really important, have to try to achieve it properly.
I want to specially thank you for your time, patience and help on this wonderful course, I will watch the video of lectures again and again to learn more from your lessons, thanks for all

**Pull request closed without merging**